### PR TITLE
Removing HTMLRender padding

### LINKED ISSUE #71: Remove default page margin in HTMLRender
# Description
Update HTMLRenderLayerNavigation to reposition title pill when at the edge of the window and remove the default margin affordance from HTMLRender.

### PR DESCRIPTION
**Pull request by williamw2**
# Pull Request

## 📖 Description

Removing the 40px padding in the HTMLRender component and adjusting the HTMLRenderNavigation logic to reposition the pill and highlight borders when the selected element is too close to the edge of the screen. Whenever the top of the target element (selected or highlighted) is less than the calculated height of the pill the pill is relocated to appear inside the highlight region. Also, normally the selected border is positioned to be outside of the selected element but this change moves the selected border to the inside if the target element is exactly adjacent to the edge of the screen. Finally, the CSS class names in the HTMLRenderNavigation component have been updated to match the style guidelines.

### 🎫 Issues

Closes #71 

### General

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.

### CODE REVIEW

File: packages/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.style.ts [hunk 1]
```diff
@@ -30,14 +30,20 @@ export const htmlRenderLayerNavigationStyles = (context, definition) => css`
         margin: calc(${FocusOutlineWidthProperty} * -1px) 0 0
             calc(${FocusOutlineWidthProperty} * -1px);
     }
-    .click-layer.active {
+    .navigation-select__insetVert {
```
Comment (janechu) [inline packages/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.style.ts:33]: I would just use the whole word, or change it to x/y:

```suggestion
    .navigation-select__insetY {
```

File: packages/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.style.ts [hunk 2]
```diff
@@ -30,14 +30,20 @@ export const htmlRenderLayerNavigationStyles = (context, definition) => css`
...
+        margin-top: 0;
+    }
+    .navigation-select__insetHorz {
```
Comment (janechu) [inline packages/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.style.ts:36]:
```suggestion
    .navigation-select__insetX {
```

**Review by janechu — APPROVED** [review]

**Review by nicholasrice — APPROVED** [review]

File: packages/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.style.ts (hunk 1 above)
Comment (williamw2) [reply packages/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.style.ts]: Done.

File: packages/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.style.ts (hunk 2 above)
Comment (williamw2) [reply packages/fast-tooling/src/web-components/html-render-layer-navigation/html-render-layer-navigation.style.ts]: And done.

**Pull request merged**